### PR TITLE
Adds the Mimesquad ERT (Serious)

### PR DESCRIPTION
**Pull request by BurgerLUA**
[Changelogs]: 
:cl: BurgerBB
add: Adds the Mime ERT, a special type of ERT with one goal: Lower the god damn noise of the station. Mime ERT are equipped with a range of special weapons and gear that are meant to non-lethally silence a person, such as a silenced pistol that shoots soporific bullets, mutetoxin grenades, and DNA syringe guns that contain the mute gene.
admin: Admins don't have to give the Mime ERT access to the armory. They spawn with everything they need in their loadout to deal with noisy heads of staff. Really only 1-2 mime ERT are needed to deal with 1 rowdy head of staff.
/:cl:
[why]: On some rounds, heads of staff can get pretty loud. Very loud. Currently the only way centcom can deal with these peoples is magically disable loudmode for headsets or microphones over comms. I think that a much funner way of dealing with these people should be introduced: The Mime ERT.

Mime ERT loadout includes:
- ERT All-Access ID
- Mime Uniform
- Mime ERT Hardsuit + Helmet
- Mime Mask
- Earmuffs
- Sunglasses
- Mime Backpack
- Special Mime Suspenders
- Mime PDA for communication
- A muzzle for ERP
- Engineer Internals + Medipen
- Bottle of Nothing
- Mime Banana
- Mime Spraycan for communication
- DNA Syringe Gun
- Soporific 10mm Silenced Pistol, with a mime firing pin.
- Extra Soporific Mag
- x4 mute syringes
- A Flashbang, except it doesn't bang, so it's just a flash.
- A mutetoxin smoke grenade

Leaders also spawn with
- 1 Soporific Sniper Rifle, with a mime firing pin.

Mimes also have some skills, abilities, and implants as well. Mimes have:
- A mindshield implant
- A stealth implant
- Knowledge of Krav Mine-ga, a martial art that's like Krav Maga, except silent.
- Fingerguns
- Mime Wall
- Mime Barrier

It is important to note that Mime ERTs cannot break their vow of silence.

### CODE REVIEW

File: code/modules/projectiles/pins.dm [hunk 1]
```diff
@@ -150,6 +150,17 @@
 	desc = "Advanced clowntech that can convert any firearm into a far more useful object. It has a small nitrobananium charge on it."
 	selfdestruct = TRUE
 
+/obj/item/firing_pin/mime
+	name = "silent firing pin"
+	desc = "An advanced firing pin that only works if the user is deadly silent."
+	color = "#FFFFFF"
+	fail_message = "<span class='warning'>...</span>"
+
+/obj/item/firing_pin/mime/pin_auth(mob/living/user)
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/pins.dm:159]: Mute trait should make it useable as well

**Review by Trilbyspaceclone — APPROVED** [review]
<3 Love this

File: code/game/objects/items/grenades/chem_grenade.dm [hunk 2]
```diff
@@ -597,6 +597,31 @@
...
+
+	beakers += B1
+	beakers += B2
+
+
+
+
+
+
+
```
Comment (deathride58) [inline code/game/objects/items/grenades/chem_grenade.dm]: Needless whitespace makes me cry

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 3]
```diff
@@ -225,10 +225,24 @@ Contains:
 	desc = "Standard issue medical suit for the ERT."
 	icon_state = "ert_medical"
 	item_state = "ert_medical"
-	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/ert/med
+	helmettype = /obj/item/clothing/suit/space/hardsuit/ert/med
```
Comment (deathride58) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: ???

**Review by deathride58 — CHANGES_REQUESTED** [review]

File: code/modules/clothing/spacesuits/miscellaneous.dm (hunk 3 above)
Comment (BurgerLUA) [reply code/modules/clothing/spacesuits/miscellaneous.dm]: don't know what happened here

**Comment by BurgerLUA** [comment]
@deathride58 Done

**Comment by BurgerLUA** [comment]
aaaa so many merge conflicts jfc

**Comment by silicons** [comment]
stale

**Pull request closed without merging**